src/organizations/components/OrgProfileTab/index.tsx

```diff
@@ -80,6 +80,7 @@ const OrgProfileTab: FC = () => {
             justifyContent={JustifyContent.SpaceBetween}
             stretchToFitWidth={true}
             style={orgProviderExists ? {width: '85%'} : {width: '48%'}}
+            testID="org-profile--labeled_data"
```

Comment:
For consistency sake, I think this should be `org-profile--labeled-data` (dash, instead of underscore)

Reply:
you're totally right.